recipes/beyla-golden-signals/beyla-daemonset.yaml

```diff
@@ -41,8 +41,9 @@ spec:
               memory: 100Mi
             limits:
               memory: 500Mi
-          image: grafana/beyla:1.2.0
+          image: grafana/beyla:1.3.3
           securityContext:
+            privileged: true
```

Comment:
Without this new line I'm seeing in the logs

`time=2024-03-05T16:20:03.490Z level=ERROR msg="couldn't trace process. Stopping process tracer" component=ebpf.ProcessTracer path=/pause pid=6748 error="attaching socket filter: operation not permitted"`

What's strange is the Beyla code hasn't changed in ~10 months https://github.com/grafana/beyla/blame/main/pkg/internal/ebpf/instrumenter.go#L211-L222

Reply:
You can re-add the privileged flag, with a TODO above it, and I can investigate.

Reply:
I don't want you to block on it

Reply:
SG thanks